src/components/Modals/SelectTokenModal/style.ts

```diff
@@ -78,6 +78,11 @@ const useStyles = makeStyles(() => ({
   paper: {
     background: 'transparent',
     boxShadow: 'none'
+  },
+  clearIcon: {
+    minWidth: 12,
+    height: 12,
+    marginLeft: 8
```

Comment:
maybe we should add a hover on icon. IMO `cursor: 'pointer'` looks better

Reply:
zrobione